src/main/java/com/example/geoIot/service/device/DeviceTrackerRedisServiceImpl.java

```diff
@@ -16,16 +18,15 @@
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 @Service
 @EnableScheduling
-public class DeviceTrackerRedisServiceImpl implements DeviceTrackerRedisService {
+public class DeviceTrackerRedisServiceImpl implements DeviceTrackerRedisService{
 
-    private final long MINUTES = 1000 * 60;
+    private final long MINUTES = 1000 * 60 *5;
```

Comment:
MINUTES constant is unintuitive with the new *5 (5 minutes) value

Reply:
"The change was made to the minutes constant.